fitbenchmarking/controllers/tests/test_controllers.py

```diff
@@ -768,6 +769,23 @@ def test_ceres(self):
             controller._status = 2
             self.shared_tests.check_diverged(controller)
 
+    def test_kmpfit(self):
+        """
+        KmpFitController: Tests for output shape
+        """
+        controller = KmpfitController(self.cost_func)
+
+        # test one from each class
+        minimizers = ['lm-lsqr']
+        for minimizer in minimizers:
+            controller.minimizer = minimizer
+            self.shared_tests.controller_run_test(controller)
+
+            controller._status = 0
```

Comment:
If you get rid of status, this being a "proper" message will be good as it tests that the parsing of the message is correct